src/libs/auth.ts

```diff
@@ -46,23 +47,30 @@ export const getOidcConfig = () => {
     stateStore: new WebStorageStateStore({ store: getLocalStorage() }),
     userStore: new WebStorageStateStore({ store: getLocalStorage() }),
     automaticSilentRenew: true,
-    // Leo's setCookie interval is currently 5 min, set refresh auth then 5 min 30 seconds to gurantee that setCookie's token won't expire between 2 setCookie api calls
+    // Leo's setCookie interval is currently 5 min, set refresh auth then 5 min 30 seconds to guarantee that setCookie's token won't expire between 2 setCookie api calls
     accessTokenExpiringNotificationTimeInSeconds: 330,
     includeIdTokenInSilentRenew: true,
     extraQueryParams: { access_type: 'offline' },
+    redirect_uri: '', // this field is not being used currently, but is expected from UserManager
   };
 };
 
 const isGoogleAuthority = () => {
   return _.startsWith('https://accounts.google.com', authStore.get().oidcConfig.authorityEndpoint);
 };
 
-const getAuthInstance = () => {
+const getAuthInstance = (): any => {
   return authStore.get().authContext;
 };
 
 export const signOut = () => {
   // TODO: invalidate runtime cookies https://broadworkbench.atlassian.net/browse/IA-3498
+  const sessionEndTime: number = Date.now();
+  Ajax({}).Metrics.captureEvent(Events.userLogout, {
```

Comment:
```suggestion
  Ajax().Metrics.captureEvent(Events.userLogout, {
```

If Ajax is given a parameter, that parameter must be an AbortSignal. Generally, we never want to cancel metrics requests, so there's no need for a signal here.

Same comment for other uses of `Ajax().Metrics.captureEvent` in this PR.